src/main/java/core/basesyntax/service/date/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.date;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private Map<String, Integer> storage;
```

Comment:
Please follow the encapsulation principle. Hide inner class elements with the help of access modifiers. It's a bad practice to make your class exposed.